src/clients/discord/discord.voice.service.ts

```diff
@@ -108,6 +109,29 @@ export class DiscordVoiceService {
         this.disconnect();
       }
     });
+
+    const voiceChannelId = channel.id;
+    const intervalId = setInterval(async () => {
```

Comment:
As mentioned in #335 , you must clear the interval. Otherwise you'll create memory leaks and unclean shutdowns.
I suggest you to hook into the Nestjs events: https://docs.nestjs.com/fundamentals/lifecycle-events#lifecycle-events-1
The clearInterval on line 119 is fine.

Reply:
You marked this as resolved but it has not been fixed yet  @Uo1428 . Please see the link above and fix it.

Reply:
Fixed